src/js/view/controls/controlbar.js

```diff
@@ -137,8 +138,12 @@ export default class Controlbar {
 
         // Do not initialize volume slider or tooltip on mobile
         if (!this._isMobile) {
-            volumeGroup = new VolumeTooltip(_model, 'jw-icon-volume', vol,
-                cloneIcons('volume-0,volume-50,volume-100'));
+            if (_model.get('audioMode')) {
+                volumeContainer = document.createElement('div');
```

Comment:
This creates a `div` for the horizontal volume slider to go in. Passing this `div` into `VolumeTooltipIcon` puts the tooltip for the icon into the `div`.